### ISSUE TITLE
constantly listen for input

### ISSUE THREAD

**Issue by jhoogeboom** [kunal732/MacLexa#10]
Can it be configured to listen for input without requiring pressing the option button?

**Comment by kunal732** [comment]
Yeah, I ultimately want to incorporate this library https://snowboy.kitt.ai/ which will do just that

**Comment by kunal732** [comment]
If anyone else knows any other libraries to add a hot word or wake word. Let me know
